Index: templates/base.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Base | Path: Unknown</title>\r\n    <link rel=\"stylesheet\" type=\"text/css\" href=\"{{ url_for('static', filename='css/normalize.css') }}\">\r\n    <link rel=\"stylesheet\" type=\"text/css\" href=\"{{ url_for('static', filename='css/style.css') }}\">\r\n</head>\r\n<body>\r\n    <header>\r\n        <!-- Header content here -->\r\n    </header>\r\n    {% block content %}\r\n    {% endblock %}\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/base.html b/templates/base.html
--- a/templates/base.html	
+++ b/templates/base.html	
@@ -5,6 +5,7 @@
     <title>Base | Path: Unknown</title>
     <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/normalize.css') }}">
     <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style.css') }}">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/Tutorial.css') }}">
 </head>
 <body>
     <header>
Index: static/css/Tutorial.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/static/css/Tutorial.css b/static/css/Tutorial.css
new file mode 100644
--- /dev/null	
+++ b/static/css/Tutorial.css	
@@ -0,0 +1,56 @@
+
+body, h1, h2, p, img {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    line-height: 1.6;
+}
+
+h1, h2 {
+    color: #333;
+}
+
+
+body {
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+
+.Introduction_text {
+    margin-top: 20px;
+}
+
+.Introduction_image {
+    max-width: 100%;
+    height: auto;
+}
+
+.Target_and_Dif {
+    margin-top: 30px;
+    display: flex;
+    justify-content: space-between;
+}
+
+
+.Game_difficulty, .Target_distance {
+    width: 48%;
+}
+
+.Game_Dif_image, .Target_Dis_image {
+    max-width: 100%;
+    height: auto;
+}
+
+.Game_Dif_description, .Target_Dis_description {
+    margin-top: 10px;
+}
+
+
+p {
+    color: #555;
+    margin-bottom: 10px;
+}
diff --git a/project.css b/project.css
deleted file mode 100644
